src/blog/2022/10/ff-docker-gcp.md

```diff
@@ -0,0 +1,180 @@
+
+Once you have reserved your IP it will be shown in the External IPv4 address field, write it down as we will need it later to create the DNS records. Our IP address was 34.125.156.130.
+
+![GCP Create VM interface](images/4.png)
+
+You are now ready to create and boot your VM, scroll to the bottom of the page and press Create. It can take a minute or two for the VM to be ready to use.
+
+# DNS Set Up
+
+So that you can run FlowForge on your newly created GCP VM you will need to set up 2 DNS records. These records are slightly different to what is suggested in the FlowForge install docs. We were keen to be able to run other services on this domain so we set up the following records.
+
+![DNS interface](images/5.png)
+
+Depending on your DNS provider it can take seconds to hours for these records to become live. Our’s were in place very quickly, You can validate the DNS records on a Mac or Linux PC using the dig command.
+
+![Dig command](images/6.png)
+
+The DNS we need is now in place.
+
+# FlowForge Docker Installation
+
+The next step is to install the FlowForge Docker build on our GCP VM. If you return to GCP you should see that your VM is now up and running, you can now click on SSH to connect to your VM. This will open up a browser based SSH session to your VM.
+
+![Access SSH in GCP](images/7.png)
+
+Once you have a Secure Shell (SSH) session open, the first step is to install Docker using the following commands.
+
+`sudo apt-get update`
+  
+	
+`sudo apt-get install \
+    		ca-certificates \
+    		curl \
+    		gnupg \
+   		lsb-release`
+      
+
+`sudo mkdir -p /etc/apt/keyrings`
+
+
+`curl -fsSL https://download.docker.com/linux/debian/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg`
+
+`echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/debian \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null`
+  
+
+`sudo apt-get update`
+
+
+`sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin`
+
+You can read a lot more detail about what each these commands actually do [here](https://docs.docker.com/engine/install/debian/).
+
+# Download FlowForge’s latest Docker build
+
+The next step is to get the codebase for FlowForge onto your VM, to do so you will need to run the following commands. Please note that we are working with our 0.10.0 build, you will need to update the version number in the commands below if you are working with a newer build.
+
+Install Wget so we can download FlowForge from Github.
+
+`sudo apt-get install wget`
+
+Use Wget to download the files we need.
+
+`wget https://github.com/flowforge/docker-compose/archive/refs/tags/v0.10.0.tar.gz`
+
+Make the directory where we will store FlowForge.
+
+`sudo mkdir /opt/flowforge`
+
+Uncompress FlowForge.
+
+`tar zxf v0.10.0.tar.gz`
+
+Copy Flowforge into the directy we created.
+
+`sudo cp -r docker-compose-0.10.0/ /opt/flowforge`
```

Comment:
`tar` can do this for you if you just add one argument `--directory /opt/flowforge`

Reply:
I tried tar zxf flowforge-0.10.0.tar.gz --directory /opt/flowforge/ but it gives me an error. Is that what you were intending?